.github/workflows/workload-cluster-aro.yml

```
@@ -268,9 +268,11 @@ jobs:
 
     - name: 'Run test-all'
       # Azure credentials and config are set at job level (see env: above)
+      # DEPLOY_CRS_TIMEOUT has a 10m buffer over DEPLOYMENT_TIMEOUT so the deployment
+      # loop can produce a graceful t.Fatalf before Go's test timer panics.
       env:
         DEPLOYMENT_TIMEOUT: "${{ env.TEST_TIMEOUT_MINUTES }}m"
-        DEPLOY_CRS_TIMEOUT: "${{ env.TEST_TIMEOUT_MINUTES }}m"
+        DEPLOY_CRS_TIMEOUT: "${{ fromJSON(env.TEST_TIMEOUT_MINUTES) + 10 }}m"
```

actionlint (1.7.12)

[error] got unexpected character '+' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)